files/unstable/https_certificate.pem

```diff
@@ -0,0 +1,63 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Are these public keys?

Reply:
Not _exactly_, but close; these are two certificates, each is a public key plus a signature.
Our own certificate is the first and is signed with Let's Encrypt intermediate key. The second one is the Let's Encrypt intermediate certificate signed with a root CA.